src/main/java/ui/Ui.java

```diff
@@ -0,0 +1,46 @@
+package ui;
+
+/**
+ * The Ui class deals with user interactions by converting the output of the chatbot
+ * to a more user-friendly output.
+ */
+public class Ui {
+    // Basic UI & messages for the chatbot to print to console
+    private static final String HORIZONTAL_LINE = "-~-~-~-~-~-~-~-~--~-~-~-~-~-~-~-~-";
+    private static final String INTRO = "Hello there! I'm Ekud. :)\n" +
+            "What can I do for you? :O";
+    private static final String OUTRO = "Goodbye, have a nice day! :p";
+
+    /**
+     * Prints a message formatted in between 2 horizontal lines for the user.
+     * @param message Text to be printed.
+     */
+    public static void printMsg(String message) {
```

Comment:
Instead of creating a generic printMsg method, i think it would be good to outline certain methods to allow the ui to print certain lines that are commonly printed (e.g. adding of tasks, marking, unmarking, deleting, etc.)